src/xunit.performance.analysis/Program.cs

```diff
@@ -287,14 +287,17 @@ private static void WriteTestResultsHtml(Dictionary<string, Dictionary<string, T
         private static string EscapeCsvString(string str)
         {
             // Escape the csv string
-            if(str.Contains("\""))
+            if (str.Contains("\""))
             {
-                str = "\"" + str.Replace("\"", "\"\"") + "\"";
+                str = str.Replace("\"", "\"\"");
             }
-            if (str.Contains(","))
+            int numOfCommas = str.Split(',').Length - 1;
```

Comment:
Nice! Will implement this. Yes, I am enclosing the string as many times as the number of commas.